### PR TITLE
Better new scene default

### PR DESCRIPTION
**Pull request by JenusL**
It has always bugged me that standard_surface isn't the default shader on Scene_Material in new Arnold scenes since Phong was not supported anymore.
One other thing that bugged me is that the default light is too dark at intensity 1. A value of PI (3.14) would equal a light output of 1. I choose a new default value a little higher than that because that is a more realistic value to use with a good view LUT. I also added a little soft shadow equal to the size of the sun.
So in conclusion, this PR does these changes on new scenes:
- Replaces Phong with `standard_surface` on Scene_Material.
- Set `distant_light.intensity` to 4
- Set `distant_light.angle` to 0.53

### CODE REVIEW

**Comment by caron** [comment]
I am not sure about changing the scene material, some people use other renderer plugins together with arnold and they may or may not translate the mental ray phong shader for compatibility. changing that shader to be arnold specific might be too much.

**Comment by caron** [comment]
actually, whatever... the default light already gets changed.

**Comment by JenusL** [comment]
Yeah exactly. The light has always been changed. It's just that when Phong was not supported anymore, SA just didn't change the Scene Material shader.
